src/locales/en/ui.yml

```diff
@@ -10,6 +10,9 @@ component:
     unfavoriteAllInGroup: Unfavorite all in group
     favoriteAllInGroup: Favorite all in group
   channelInfo:
+    stats: Stats
```

Comment:
You could probably reuse the same translations in lines 16-18 below.
```
    videoCount: '{0} Videos'
    clipCount: '{n} Clips'
    subscriberCount: '{n} Subscribers'
```

Check out view/Channels.vue on how to use them